src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/common/actions/haflow/BaseHaFlowPathRemovalAction.java

```diff
@@ -85,7 +85,7 @@ protected void removeRejectedPaths(Collection<PathId> subPathIds, Collection<Pat
             flowPathRepository.remove(subPathId)
                     .ifPresent(subPath -> {
                         updateIslsForFlowPath(subPath);
-                        HaFlowHistoryService.using(carrier).save(HaFlowHistory.withTaskId(correlationId)
+                        FlowHistoryService.using(carrier).save(HaFlowHistory.of(correlationId)
```

Comment:
The same here

Reply:
I created a local variable for the flow history service.